packages/atlas/src/components/_video/VideoListItem/VideoListItem.styles.ts

```diff
@@ -0,0 +1,78 @@
+import styled from '@emotion/styled'
+
+import { Loader } from '@/components/_loaders/Loader'
+import { cVar, media, sizes } from '@/styles'
+
+export const Wrapper = styled.div<{ variant: 'small' | 'large'; clickable?: boolean }>`
+  width: 100%;
+  display: flex;
+  gap: ${sizes(4)};
+  flex-direction: column;
+  position: relative;
+  overflow: hidden;
+  cursor: pointer;
+  padding: ${sizes(3)} ${sizes(4)};
+
+  ${media.xs} {
+    flex-direction: row;
+
+    > *:first-child {
```

Comment:
Could we avoid using first-child here and replace it with first-of-type? Not a big deal, but emotion is complaining about this: 
![image](https://user-images.githubusercontent.com/51168865/212749245-6635d8c7-30f6-4587-99a9-6f7ba73aa0d4.png)